toolkits/global/packages/global-card/demo/context.json

```diff
@@ -1,11 +1,80 @@
 {
-	"image": {
-		"src": "https://picsum.photos/500/500",
-		"alt": "descriptive alternative text"
+	"cardsDefault": {
```

Comment:
In this demo json config file could we portray some data that defines some of the variables used in the hbs template i.e. `noList`, `noListItem`, `level` and `aspectRatio`. I know they are mentioned in the README and in the hbs template but I think it will help consumers understand the implementation better if they can see example data that implements those. Particularly in understanding and tracing `{{>card parent=..}}` in the demo hbs

Reply:
@benjclark Good idea! Thanks. I'll do separate demos for each.